airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/InlineEnrollmentCallout.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { PropsWithChildren } from "react";
+import { FormattedMessage } from "react-intl";
+
+import { Callout } from "components/ui/Callout";
+import { Text } from "components/ui/Text";
+
+import styles from "./InlineEnrollmentCallout.module.scss";
+
+export const EnrollLink: React.FC<PropsWithChildren<unknown>> = ({ children }) => {
+  const onEnrollClick = () => {
+    return null;
+  };
+
+  return (
+    <span
```

Comment:
Ideally this should just be a `button` element that's styled to look as we need it, i.e. no background, border, inline style. We're doing a lot of work to try simulate a button here, and will always be a bit short of really doing it, e.g. at the moment the button doesn't trigger on <kbd>Space</kbd> keys, which is the default behavior a button usually has. Making this a `<button>` and style it to look like we need is usually the preferred (and easier) way.

Additionally: Currently the `onEnrollClick` doesn't do anything. What it this eventually planned to do? Show a modal or navigating away? (Because in case it's navigating away, this should ideally be a `a` instead.)

Reply:
Show a modal, which is why it's not an a.

Reply:
changed to a button in https://github.com/airbytehq/airbyte/pull/21315/commits/52c7b32460f26fa0bb1ee9ac1e233a30eaaa71af